### ISSUE TITLE
module.js still contains .value("Firebase", exports.Firebase);

### ISSUE THREAD

**Issue by richtera** [firebase/angularfire#808]
The latest master commit of module.js still has
```js
.value("Firebase", exports.Firebase);
```
This should probably be changed to
```js
.value("firebase", exports.firebase);
```

**Comment by davideast** [comment]
Good call @richtera. We're doing a release for #799 soon and I'll make sure to add that in there.

**Comment by davideast** [comment]
Closed as of #841
